packages/backend/src/telemetry/privacy.test.ts

```diff
@@ -0,0 +1,95 @@
+import { privacy } from './privacy'
+describe('Privacy functions', () => {
+  const clipParams = {
+    minBucketSize: 1000,
+    maxBucketSize: 10000
+  }
+
+  afterEach(() => {
+    jest.unmock('./privacy')
+  })
+
+  test('generateLaplaceNoise should return a different number each time', () => {
+    const scale = 0.5
+    const noise1 = privacy.generateLaplaceNoise(scale)
+    const noise2 = privacy.generateLaplaceNoise(scale)
+    expect(noise1).not.toBe(noise2)
+  })
+
+  test('computePrivacyParameter should return 0 when sensitivity is 0', () => {
```

Comment:
we should add a non-0 test here

Reply:
done. I hope that's what you meant by it